tests/button/Button.tsx

```diff
@@ -1,16 +1,14 @@
 import * as React from 'react';
 
-interface ButtonProps { };
-
-export class Button extends React.Component<ButtonProps, void>{
+export class Button extends React.Component {
```

Comment:
Why are we removing the interface here? This is the only "typescript" that should be transpiled completely away - I'd be very wary of trusting the line numbers if this was removed.

Reply:
Fair enough.

Reply:
I didn't think about that. I had removed the types because they were showing an error and then removed the interface because it wasn't being used.

That said, the target for this test is `es5` so the multiline comment should get squished and result in different line numbers, right?

Reply:
It seems you're right:
![screen shot 2017-10-03 at 7 36 00 pm](https://user-images.githubusercontent.com/12202757/31135521-1f6fbce2-a872-11e7-8c88-685f5c5180f6.png)

Reply:
Still, I can just imagine that sometime in the future we'll end up getting wrecked by something we think is transpiling properly, but that the lines number matching is just accidental.

I'd definitely like to keep a typescript-only construct in there - if nothing else then for my own peace of mind.

Reply:
Ok .. I've pushed this change in but seem to have messed something git related

Reply:
the pr seemed kinda wonky right after your last push, but I think it's okay to me now. Nice touch with the comment.

Reply:
yeah .. there were some conflicts with the master